website/views/campaigns.py

```diff
@@ -29,13 +29,18 @@
 
 @campaigns_bp.route("/")
 def list_campaigns():
+    current_user = None
+
+    if g.session_data is not None:
```

Comment:
Ditto - drop the `is not None`?

(Unless we want this to run for a falsy value of `g.session_data`)